public/images/Svg.js

```diff
@@ -30,7 +30,10 @@ module.exports = function(name, props) { // eslint-disable-line
   if (!svg[name]) {
     throw new Error("Unknown SVG: " + name);
   }
-  let className = props ? `${name} ${props.className}` : name;
+  let className = name;
```

Comment:
Maybe the better solution here would be for me to use the `classnames` lib like the rest of the app uses?